cmake/fw_zip.cmake

```diff
@@ -20,8 +20,7 @@ function(generate_dfu_zip)
   endif()
 
   add_custom_command(
-    TARGET ${GENZIP_TARGET}
-    POST_BUILD
+    OUTPUT ${GENZIP_OUTPUT}
```

Comment:
There are no targets depending on this output, which means `${GENZIP_OUTPUT}` will not be generated.
For example `dfu_application.zip`

This was probably the reason why a post build step were used.

Reply:
Added a custom target which is adds the zip generation to the default target.